Makefile

```diff
@@ -16,8 +18,19 @@ DOCKERCOMPOSESETUPDACMOCKL1 := dac-setup-committee-mock-l1
 DOCKERCOMPOSESETUPDACSEPOLIA := dac-setup-committee-sepolia
 DOCKERCOMPOSEEXPLORERL1 := cdk-validium-explorer-l1
 DOCKERCOMPOSEEXPLORERL1DB := cdk-validium-explorer-l1-db
+DOCKERCOMPOSEEXPLORERFRONTENDL1 := explorer-frontend-l1
+DOCKERCOMPOSEEXPLORERSTATSL1 := explorer-stats-l1
+DOCKERCOMPOSEEXPLORERSTATSDBL1 := explorer-stats-db-l1
+DOCKERCOMPOSEEXPLORERSMARTCONTRACTVERIFIERL1 := explorer-smart-contract-verifier
```

Comment:
> but the service is actually named explorer-smart-contract-verifier. so the URL has to be changed

Nice catch. Fixed.

> why there is no L2 explorer equivalent of this? is it b/c it can be shared? if that's case, we should also rename it.

You're right. sig-provider, sc-verifier, visualizer are shared services. Renamed respectively (l1 suffix removed).